src/Symfony/Component/Security/Core/Authorization/TraceableAccessDecisionManager.php

```diff
@@ -48,7 +48,7 @@ public function __construct(AccessDecisionManagerInterface $manager)
     /**
      * {@inheritdoc}
      */
-    public function decide(TokenInterface $token, array $attributes, $object = null): bool
+    public function decide(TokenInterface $token, array $attributes, $object = null/*, bool $allowMultipleAttributes = false*/): bool
```

Comment:
we need to also explicitly add the `@param` annotation to opt-out from the deprecation that will be triggered otherwise.

Reply:
added